ecosystem/typescript/sdk/.prettierrc

```diff
@@ -1,6 +1,6 @@
 {
   "semi": true,
   "trailingComma": "all",
-  "singleQuote": false,
```

Comment:
@jjleng just saw this... we had discussions around standards- can we revert + make sure in the future to involve rest of team?

Reply:
No problem. Double quotes are fine with me. But the typescript linter complains double quote strings. I can remove that linter. But what are the reasons for insisting on double quotes?

Reply:
FYI, our web-wallet also uses single quotes.

Reply:
@CapCap removing this is a non-op since the default for that setting is anyways false

Reply:
I also prefer double quotes. Makes it easier to switch between languages since most (in particular Rust) use them as well.

Reply:
I actually don't have strong opinions about the quotes. But I think we should have a broader consensus on this. Our web-wallet repo also uses the airbnb typescript linter and single quotes. We should reach an agreement and be consistent.

Reply:
yeah agree, with that. Imo we should move the prettierrc and eslint config to the repo root.

Reply:
@hariria what's your opinion?